docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+Upload Rights
+#############
+
+Those who are allowed to upload the kernel to the Ubuntu archive take their
```

Comment:
Hi @gagath , I think this sentence should be reworded to indicate the impact / importance of the role instead of referencing how team members (are assumed) to _feel_ about the role.

The original Wiki doesn't put it this way either.

```suggestion
Those who are allowed to upload the kernel to the Ubuntu archive have a serious responsibility.
```